src/engine.rs

```diff
@@ -81,6 +89,9 @@ mod tests {
     use super::*;
     use std::path::Path;
 
+    // Arbitrary, used to verify that the runner works as expected.
+    const RUNTIME_THRESHOLD: Duration = Duration::from_millis(5);
```

Comment:
Let's work that comment into the name. `ARBITRARY_THRESHOLD` should do it. Naturally any test is used to make sure the runner works as expected.